basiclibrary/src/main/java/basiclibrary/Library.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basiclibrary;
+
+import java.lang.reflect.Array;
+import java.util.Random;
+
+public class Library {
+    public boolean someLibraryMethod() {
+        return true;
+    }
```

Comment:
After you have created your project and run the test for this method to verify it's working... you can safely delete this method and its test.